scripts/check-binaries.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function test_command() {
+    RED="\033[01;31m"
+    RESET="\033[0m"
+
+    if ! command -v "${1}"; then
+        echo -e "${RED}${1} not found!${RESET}"
+        exit 1
+    fi
+}
+
+BINARIES=( "clang" "ld.lld" "llvm-ar" "llvm-nm" )
+for BINARY in "${BINARIES[@]}"; do
+    test_command "${BINARY}-${LLVM_VERSION}"
+done
+
+QEMU_PREFIXES=( "arm" "aarch64" "i386" "mips" "mipsel" "ppc" "ppc64" "x86_64" )
+for QEMU_PREFIX in "${QEMU_PREFIXES[@]}"; do
```

Comment:
technically these are suffixes; the binutils ones below are prefixes.

Reply:
Done, thanks!